### PR TITLE
added layout

### PR DESCRIPTION
**Pull request by naberalgamma**

### CODE REVIEW

File: frontend/package.json [hunk 1]
```diff
@@ -5,7 +5,6 @@
   "dependencies": {
     "@emotion/react": "^11.9.0",
     "@emotion/styled": "^11.8.1",
-    "@mui/icons-material": "^5.6.0",
```
Comment (steadyGuy) [inline frontend/package.json:8]: Honestly saying, you need somehow to setup this package. Maybe try `yarn add @mui/material-icons --network-timeout 500000` (setup big timeout for installation).

File: frontend/src/App.tsx [hunk 2]
```diff
@@ -1,13 +1,17 @@
...
+import NavBar from 'Header/NavBar';
+import Footer from 'Footer/Footer';
 
 function App() {
   return (
     <BrowserRouter>
+      <NavBar />
       <Routes>
-        <Route path="/" element={<HomeScreen />} />
+        <Route path="/Map" element={<Map />} />
```
Comment (steadyGuy) [inline frontend/src/App.tsx]: I think tat the path should be "/" insted of "Map". But we need to discuss it with team.

File: frontend/src/App.tsx [hunk 3]
```diff
@@ -1,13 +1,17 @@
...
+import Map from 'components/Map/index';
+import NavBar from 'Header/NavBar';
+import Footer from 'Footer/Footer';
 
 function App() {
   return (
     <BrowserRouter>
+      <NavBar />
       <Routes>
-        <Route path="/" element={<HomeScreen />} />
```
Comment (steadyGuy) [inline frontend/src/App.tsx:9]: And why you remove HomeScreen? I thought that we decided to structure our logic as pages (views in our case) and components inside these views.

**Review by steadyGuy — CHANGES_REQUESTED** [review]

File: frontend/src/App.tsx (hunk 3 above)
Comment (dobbi-free) [reply frontend/src/App.tsx:9]: I agree with it. So Map must be in HomePage, and then HomePage in the route.

**Comment by dobbi-free** [comment]
Replace the component of your Map with a Map from the develop branch

**Review by steadyGuy — APPROVED** [review]

**Review by steadyGuy — APPROVED** [review]
Fix conflicts before merge

File: frontend/src/App.tsx [hunk 4]
```diff
@@ -1,13 +1,19 @@
...
+import NavBar from 'components/Header/NavBar';
+import Footer from 'components/Footer/Footer';
 
 function App() {
   return (
     <BrowserRouter>
+      <NavBar />
       <Routes>
         <Route path="/" element={<HomeScreen />} />
+        <Route path="/Map" element={<Map />} />
```
Comment (romawht22) [inline frontend/src/App.tsx]: Why do we need this route?

Comment (naberalgamma) [reply frontend/src/App.tsx]: ok i can remove it

**Review by romawht22 — APPROVED** [review]

**Pull request merged**